templates/data-sources/provisioner.md.tmpl

```diff
@@ -0,0 +1,28 @@
+---
+Title: "Provisioner data source in a management cluster"
+Description: |-
+  Provisioner data source in a management cluster.
+---
+
+# Management Cluster Provisioner
+
+Read provisioner in a management cluster using this Terraform module.
+
+### Tanzu Kubernetes Grid Service and Tanzu Kubernetes Grid vSphere
+- Only provisioner read feature is enabled for Tanzu Kubernetes Grid Service and Tanzu Kubernetes Grid vSphere management clusters.
```

Comment:
Since it is datasource its just read operation for all so this statement specific TKG flavours maybe redundant here.